keystone/baseline.py

```diff
@@ -110,6 +110,15 @@ def tightWindow(spectrum, spaxis,
         mask[(spaxis < (v0 - outerwindow))] = True
     return(~mask)
 
+def clipper(file_in, clip_lr=[100,100]):
```

Comment:
Left/right doesn't seem like a great description here since velocity vs. frequency can change this.  Can we also split the keywords instead of passing the list?

Reply:
I have changed clip_lr to lowFreq and highFreq to make it more explicit.